data/frame.go

```diff
@@ -1,21 +1,27 @@
-// Package data provides data structures that Grafana recognizes. The Frame
-// object represents a Grafana Dataframe which can represent data such as tables
+// Package data provides data structures that Grafana recognizes.
+//
+// The Frame object represents a Grafana Dataframe which can represent data such as tables
 // and time series.
+//
+// Frames can be encoded using Apache Arrow (https://arrow.apache.org/) for transmission.
+//
+// The corresponding Grafana frontend package the @grafana/data package
+// (https://github.com/grafana/grafana/tree/master/packages/grafana-data).
```

Comment:
@ryantxu is there a better link for the above?

Reply:
This is good for now -- before 7.0 the generated docs will get published, and I think that will be more appropriate.